### DISCUSSION TITLE
设置多语言的开关

### DISCUSSION THREAD

**Discussion by zha-ji-tui** [soybeanjs/soybean-admin#520] [Q&A]
希望能加上多语言开关，目前要删除多语言得动很多地方的配置

**Comment by Azir-11** [comment]
多条分支维护起来比较费精力，i18n又是不能放弃的内容，暂不考虑了，你可以隐藏语言切换按钮，后续新的代码只写中文

**Reply by zha-ji-tui** [reply]
有空我来抽离一个分支吧

**Reply by Azir-11** [reply]
> 有空我来抽离一个分支吧

你能做到commit级别的跟进更新吗？

**Reply by zha-ji-tui** [reply]
只是在你的基础版本上抽离，供有需求的人进行项目初始化。不过如果后续有什么新的改动或者依赖加入进来可能就无法实时跟进了

**Reply by Azir-11** [reply]
> 只是在你的基础版本上抽离，供有需求的人进行项目初始化。不过如果后续有什么新的改动或者依赖加入进来可能就无法实时跟进了

不能跟进就没什么意义了，最近要做的其他事情不少，暂时不考虑

**Comment by honghuangdc** [comment]
可以文档上体现快速关闭多语言的步骤
